src/router/index.js

```diff
@@ -0,0 +1,27 @@
+import Vue from 'vue'
+import VueRouter from 'vue-router'
+import HomeView from '../views/MoviesPageView.vue'
+import ActorView from '../views/ActorPageView.vue'
+
+Vue.use(VueRouter)
+
+const routes = [
+  {
+    path: '/',
+    name: 'home',
+    component: HomeView
```

Comment:
Use [Lazy Loading](https://router.vuejs.org/guide/advanced/lazy-loading.html) for router view components.